### DISCUSSION TITLE
[Feature] Support go client basic version in HugeGraph

### DISCUSSION THREAD

**Discussion by izliang** [apache/incubator-hugegraph#2384] [Show and tell]
The hugegraph client basic version in go language has been released, which includes basic gremlin, propertykey, vertexLabel, edgeLabel, Vertex APIs, and some Edge APIs. It can use simple functions and interact with hugegraph server.
Welcome everyone to jointly build the go language version of the client. Here is the readme file for hugegraph client go. Welcome everyone to contribute the code. [(Click me to readme)](https://github.com/apache/incubator-hugegraph-toolchain/tree/master/hugegraph-client-go#readme)

**Comment by liuxiaocs7** [comment]
Could we divide it into smaller subtasks like [python-client](https://github.com/apache/incubator-hugegraph-ai/issues/2) and continue to track this task in [this issue](https://github.com/apache/incubator-hugegraph-toolchain/issues/512)?
